release/stage-release.sh

```diff
@@ -8,14 +8,21 @@ if ! command -v releaser >/dev/null; then
   exit 1
 fi
 
+function missing_env_var() {
+  echo "Error: Environment variable ${1} is not set." >&2
+  exit 1
+}
+
 function usage() {
   echo "Usage: ${0} VERSION" >&2
   exit 1
 }
 
 [ ${#} -eq 1 ] || usage
+[ -z "${CK8S_GITHUB_TOKEN:-}" ] && missing_env_var "CK8S_GITHUB_TOKEN"
```

Comment:
This is a nit as I don't like the free `&&`s because they are an exception to the usual rules we have in bash with pipefail. :sweat_smile: 

```suggestion
[ -n "${CK8S_GITHUB_TOKEN:-}" ] || missing_env_var "CK8S_GITHUB_TOKEN"
```